tools/runners/run-all.js

```diff
@@ -70,11 +70,18 @@ class Runner {
       proxyToHost: appHost,
       onFailure
     });
+    
+    buildmessage.capture(function () {
+      self.projectContext.resolveConstraints();
+    });
+
+    const packageMap = self.projectContext.packageMap;
+    const hasMongoDevServerPackage = packageMap && packageMap.getInfo('mongo-dev-server') != null;
```

Comment:
Thanks @zimme, it's probably okay to leave it then. I personally prefer to not use lenient equality / non-equality checks on `null` / `undefined`, as when going back over old code I always end up questioning whether the leniency was intended or not ... 🙂.